### ISSUE TITLE
Ignore XHR requests

### ISSUE THREAD

**Issue by fedekau** [fedekau/ember-service-worker-race#1]
The current implementation caches every GET request made, even the ones done by the app to load data (XHR requests).

May be an option could be added to ignore the requests made to certain URLs, but it would much better that the service worker could identify those requests automatically and don't cache them.

I came to this http://stackoverflow.com/questions/40583334/how-to-detect-whether-url-request-is-ajax-in-service-worker-fetch-api but I am not sure it would work 🤔

**Comment by miguelcobain** [comment]
I'd perfectly happy with a simple regex (or multiple ones) i could specify, in which this service worker approach would be working on.
Data requests are typically under a `/api*` url.
Ideally, a blacklist and/or whitelist approach could be used.

**Comment by fedekau** [comment]
Yeah that should do the work, I will start implementing it as soon as I have some free time 😄
